test/js/components/utils/useForm.test.jsx

```diff
@@ -16,14 +16,21 @@ afterEach(() => {
 describe('useForm', () => {
   test('creates a new object', () => {
     const { result } = renderHookWithRedux(() =>
-      useForm({ fields: { testing: '' }, objectType: 'test-type' }),
+      useForm({
+        fields: { testing: '', test: true },
+        objectType: 'test-type',
+      }),
     );
-    result.current.handleSubmit({ preventDefault: jest.fn() });
+    result.current.handleSubmit({
+      preventDefault: jest.fn(),
+      target: { type: 'checkbox', checked: true },
```

Comment:
@jgerigmeyer this test doesn't seem to cover accepting checkboxes in `useForm`.

Reply:
Right -- this is calling `handleSubmit` instead of `handleInputChange`, which is I think what you're trying to test?